src/js/main.js

```diff
@@ -1,9 +1,8 @@
 export function setupForm () {
   document.addEventListener("DOMContentLoaded", function (e) {
     var triggeringHash = "#get-in-touch";
-    var formContainer = document.getElementById("contact-form-container");
+    var formContainer = document.querySelector(".contact-form-container");
```

Comment:
Why switching from ID to class?

Reply:
Beacuse of the module styles, so I've seen in other projects that we usually wrap the styles in a selector (which cannot be an ID because linting fails) e.g. https://github.com/canonical-web-and-design/global-nav/blob/main/src/sass/global-nav.scss#L20, is this necessary? if not then I can revert it to an ID